install/k8s/install.sh

```diff
@@ -17,8 +17,10 @@ fi
 k8sversion=$($kubectl version --short | grep "Server Version")
 if [[ "$k8sversion" == *"v1.4"* ]] || [[ "$k8sversion" == *"v1.5"* ]]; then
 	k8sfolder="k8s1.4"
-else
+elif [[ "$k8sversion" == *"v1.6"* ]] || [[ "$k8sversion" == *"v1.7"* ]]; then
```

Comment:
That's a good point. I had followed the model of having a separate folder, but if nothing is different, I agree with you.

Reply:
While it's true that the files in install/k8s/k8s1.6 and k8s1.8 are the same, the files in cluster/k8s/k8s1.6 and k8s1.8 are not the same, so it seems weird to me to in one case point to 1.6 files for 1.8, and in another case, point to 1.8 files.

Reply:
Would it make sense to rename install/k8s/k8s1.6 to install/k8s/rbac and install/k8s/k8s1.4 to install/k8s/pre-rbac? The primary difference between the two versions is that RBAC was added post 1.4. Cluster scripts install a specific version and are more version specific rather that being related to a featureset like the yaml files.

Reply:
It's an interesting idea, but I'm envisioning what that would look like with the next big feature set. In the future, we'd add another feature, create a new directory, but it won't be obvious that rbac is included with the new feature. It won't be obvious that it is cumulative, in other words. Would it be better to make the k8s1.8 directory a sim link to k8s1.6?

Reply:
Talked with Neelima. Here is the plan: we'll leave the k8s1.4 directory intact, as it will go away soonish. The k8s1.6 directory will be renamed rbac, and 1.6, 1.7 and 1.8 will use this directory.